index.bs

```diff
@@ -12438,6 +12436,12 @@ object is associated with.
             1.  [=list/iterate|For every=] [=LegacyWindowAlias identifier|identifier=] |id| in
                 [{{LegacyWindowAlias}}]'s [=LegacyWindowAlias identifier|identifiers=]:
                 1.  Perform [=!=] [$CreateMethodProperty$](|target|, |id|, |interfaceObject|).
+        1.  If the |interface| is declared with a [{{NamedConstructor}}] [=extended attribute=],
+            then:
+            1.  [=list/iterate|For every=] [=NamedConstructor identifier=] |id| on |interface|:
```

Comment:
I'm prety sure there can only be on NamedConstructor identifier, so this loop is not necessary.

Reply:
Per spec there can be many.  https://heycam.github.io/webidl/#NamedConstructor explicitly says "Multiple [NamedConstructor] extended attributes may appear on a given interface.".

Now in practice no one does that for the uses we have for NamedConstructor right now, so maybe we should simplify things...

Reply:
Filed https://github.com/heycam/webidl/issues/653.